spec/tapioca/compilers/dsl/sidekiq_worker_spec.rb

```diff
@@ -85,13 +85,13 @@ def perform(customer_id)
         # typed: strong
 
         class NotifierWorker
-          sig { params(customer_id: Integer).returns(String) }
+          sig { params(customer_id: ::Integer).returns(String) }
           def self.perform_async(customer_id); end
 
-          sig { params(interval: T.any(DateTime, Time), customer_id: Integer).returns(String) }
+          sig { params(interval: T.any(DateTime, Time), customer_id: ::Integer).returns(String) }
```

Comment:
Hmm, this is interesting. I would have expected this to have been fixed too.

Reply:
The reason these aren't changing is because they're hard-coded in the code: https://github.com/Shopify/tapioca/blob/main/lib/tapioca/compilers/dsl/sidekiq_worker.rb#L59.

The `Integer` param is derived from the live ruby code:
https://github.com/Shopify/tapioca/blob/main/lib/tapioca/compilers/dsl/sidekiq_worker.rb#L59

I wonder if there should be some helper that allows us to specify these manual types via code and have it get compiled to a string.